lib/hayfork/statement.rb

```diff
@@ -128,5 +128,15 @@ def reflection_for(association)
       Hayfork.reflection_for(model, association)
     end
 
+    def types_method
+      before_rails61? ? :types : :klass
```

Comment:
I'd expect this to work — does it?
```suggestion
      ActiveRecord::VERSION < "6.1" ? :types : :klass
```

Reply:
That didn't work, but `ActiveRecord.version < Gem::Version.new("6.1")` did. 🙂